<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CountDown</class>
 <widget class="QDialog" name="CountDown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Count down</string>
  </property>
  <widget class="QDialogButtonBox" name="dbbCountDown">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbHours">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbMinutes">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbSeconds">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>90</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QLabel" name="HoursLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Hours</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>sbHours</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="MinutesLbl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Minutes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>sbMinutes</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="SecondsLbl">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Seconds</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buddy">
    <cstring>sbSeconds</cstring>
   </property>
  </widget>
  <widget class="QRadioButton" name="CountDownNothing">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>102</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Do Nothing</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="CountDownSound">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>102</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Play Sound</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="CountDownProgram">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Launch Program</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCDSound">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SoundFileLbl">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>271</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="CountDownProgramCLI">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Program</string>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="placeholderText">
    <string extracomment="Program">Program</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="CountDownProgramARG">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Argument</string>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="placeholderText">
    <string extracomment="Arguments">Arguments</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="CountDownProgramWorkingDir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="placeholderText">
    <string>Working Directory</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sbHours</tabstop>
  <tabstop>sbMinutes</tabstop>
  <tabstop>sbSeconds</tabstop>
  <tabstop>CountDownNothing</tabstop>
  <tabstop>CountDownSound</tabstop>
  <tabstop>pbCDSound</tabstop>
  <tabstop>CountDownProgram</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>dbbCountDown</sender>
   <signal>accepted()</signal>
   <receiver>CountDown</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbCountDown</sender>
   <signal>rejected()</signal>
   <receiver>CountDown</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
